config/settings.py

```diff
@@ -26,7 +26,11 @@
 DEBUG = True
 
 ALLOWED_HOSTS = []
-
+#------------File_uploads_requirements------------
+import os
+MEDIA_URL = '/media/'
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
+#--------------------------------------------
```

Comment:
Why don't we get these from util_constants.py ?

Reply:
Since we are using media configuration settings in settings.py file only, we are not importing/using these settings across any other files. So, we can leave as it is Thulasi.